presto-docs/src/main/sphinx/release/release-0.137.rst

```diff
@@ -5,15 +5,18 @@ Release 0.137
 General Changes
 ---------------
 
+* Fix ``current_date`` to return correct results for all time zones.
 * Add :func:`bit_count`, :func:`bitwise_not`, :func:`bitwise_and`,
   :func:`bitwise_or`, and :func:`bitwise_xor` functions.
 * Add :func:`approx_distinct` aggregation support for ``VARBINARY`` input.
-* Fix ``current_date`` to return correct results for all time zones.
-* Improve expression optimizer to remove some redundant operations.
-* Allow using double input for :func:`approx_percentile` with an array of
-  percentiles.
+* Fix invalid plans when scalar subqueries use ``GROUP BY``, ``DISTINCT`` or ``JOIN``.
 * Do not allow creating views with a column type of ``UNKNOWN``.
 * Add create time to query detail page in UI.
+* Add support for parametric ``VARCHAR`` type.
```

Comment:
I doubt many people know what "parametric ``VARCHAR``" is but I bet most people know what `VARCHAR(lenth)` is